docs/configGuide/YAML-Config-Syntax.md

```diff
@@ -0,0 +1,511 @@
+# Detailed Arena Config Guide
```

Comment:
I think this can be combined with the Background-YAML file

Reply:
It would be too long as a doc. Our aim is to make each doc a light read and organised.